campaigns/guides/campaignFaqs.md

```diff
@@ -21,3 +21,6 @@ It is possible to have a campaign that is in 'ACTIVE' status from the TCR perspe
 
 ## 4. My campaign has been rejected by one or more carriers, how can I get them to approve?
 At this point, the carriers campaign review rules are unknown to Bandwidth. To get your campaign re-reviewed, you will have to update your campaign and re-submit.
+
+## 5. How do I share my campaigns with Bandwidth DCA?
+You can share your campaigns with Bandwidth DCA in the TCR UI or by making a request to PUT csp-api.campaignregistry.com/campaign/{campaignId}/sharing/BANDW
```

Comment:
I'm guessing this API endpoint is documented somewhere else right?

Reply:
Or is it just do the put request and you're set?

Reply:
That's a good question - customers that are importing serve as their own CSP which means they'll already be working with TCR and their APIs. TCR has their own swagger page https://csp-api.campaignregistry.com/v1/restAPI so they should already have access to all of this

Reply:
Ah ok so that's an independent service from Bandwidth. Yeah adding that link may be helpful just to clarify that this isn't something we own